data/mapping/mapping.json

```diff
@@ -4,31 +4,25 @@
       "analyzer":{
         "gp_name_analyzer":{
           "type":"custom",
+          "tokenizer":"name_tokenizer",
+          "filter":[ "lowercase", "gp_title_stop" ]
+        },
+        "surgery_name_analyzer":{
+          "type":"custom",
           "tokenizer":"standard",
-          "filter":[
-            "lowercase",
-            "gp_stop"
-          ]
+          "filter":[ "lowercase" ]
+        }
+      },
+      "tokenizer": {
+        "name_tokenizer": {
+          "type": "pattern",
+          "pattern": "[^a-zA-Z']+"
         }
       },
       "filter":{
-        "gp_stop":{
+        "gp_title_stop":{
           "type":"stop",
-          "stopwords": [ 
-            "and",
-            "the",
-            "&",
-            "doctor",
-            "doctors",
-            "dr",
-            "drs",
-            "partners",
-            "practice",
-            "surgery",
-            "medical",
-            "health",
-            "centre",
-            "center" ]
+          "stopwords": [ "doctor", "doctors", "dr", "drs" ]
```

Comment:
Seems fair enough to add possible alternatives, however, I've not seen anything other than `dr` in the name. Searched via:
* `jq '.[].doctors' data/input/gp-data-merged.json | grep <search_term>`
The only `<search_term>` returning results is `dr`.

Does the punctuation get removed from terms like `dr.` so just adding `dr` is going to cover it? The reason I ask is there 1625 results for `jq '.[].doctors' data/input/gp-data-merged.json | grep 'Dr\.' | wc -l`